docs/startleft-processors/diagram/Visio-TrustZones-Mapping.md

```diff
@@ -0,0 +1,210 @@
+can then perform more or less complex operation with that representation model.
+
+This would be an example of a Visio diagram and the simplified model we generate:
+
+![img/shapes-representation.png](img/shapes-representation.png)
+
+The shapely object we use for the Visio shapes representation is the Polygon and an extensive manual of how to manipulate 
+that objects can be found [here](https://shapely.readthedocs.io/en/stable/manual.html).
+
+## TrustZone identification
+
+---
+We have two ways to identify TrustZones in a Visio diagram that are described below. However, it is important to notice 
+that, regardless the type of TrustZone calculation, **no TrustZone will be included in the OTM if it does not appear in 
+the mapping file**. If some TrustZone candidate is identified in the Visio diagram, but it is not in the mapping file, 
+the TrustZone will be ignored or processed as a normal component depending on the case.
+
+### Parent Shape TrustZone
+**One shape that has no parent in the Visio diagram and that appears in the mapping file is considered a TrustZone**. 
+It is the case of this example:
+
+![img/parent-shape-trustzone.png](img/parent-shape-trustzone.png)
+
+The TrustZone identification is based on the parent calculation. Notice that a component may belong directly to a 
+TrustZone, but also be nested into another component. Anyway, each shape must have a parent, so, for each of them, we 
+perform the following steps:
+
+1. We check if the shape is contained into another. If so, that shape is its parent.
+2. If a shape is not contained in any other, there are two options:
+    1. If it is mapped as a TrustZone in the mapping file, it is a TrustZone.
+    2. If not, it is a component whose parent is the default TrustZone.
+
+As stated before, Visio does not provide information about if a shape is over or “belongs to“ another one. To solve this, 
+we use the shapely’s `Polygon` representations of each shape. For each shape, we check if it is contained in another one 
+by using the shapely’s `Polygon`'s function `contains(Polygon) -> bool`. This function let us know if some shape contains 
+another.
+
+#### Nested components
+Nested components are a special case where more than one shapes will return True when calling the `contains` function.
+
+![img/nested-components.png](img/nested-components.png)
+
+In the example above, the `Public Cloud` and `Custom VPC` contain the `Some ec2` shape. In this case we use the `Polygon`'s 
+function area. The parent shape is the one with the smallest area, that is, the `Custom VPC`. Then, the Custom VPC is only 
+contained in the `Public Cloud`, so it is selected as its parent. Finally, the `Public Cloud` is not contained by any 
+shape, and it is mapped as TrustZone in the mapping file, so it is converted into one.
+
+### Boundary TrustZone
+> **Note**: The tangent would be more precise than the secant, but given its center point is more difficult from the Visio 
```

Comment:
It is not clear if this means that it is more difficult to calculate, to extract or something different thing.

Reply:
Yes, sounds like the word "extract" is missing there. Changed in the review branch.